README

```diff
@@ -32,7 +30,7 @@ To get good API documentation, run:
 
     $ epydoc -n "PyFCP API manual" -o html fcp
 
-When you install this package (refer INSTALL), you should 
+When you install this package (refer INSTALL), you should
```

Comment:
I’m not a friend of how you mix whitespace changes and unrelated file removals in the same commit.